indico/modules/rb_new/client/js/common/timeline/DailyTimelineContent.jsx

```diff
@@ -58,7 +59,8 @@ export default class DailyTimelineContent extends React.Component {
         lazyScroll: null,
         minHour: 6,
         maxHour: 22,
-        hourStep: 2
+        hourStep: 2,
+        showUnused: false
```

Comment:
maybe better to default to true?

Reply:
I thought of using false because that was the initial design of the calendar and it was added as an option in the issue but both ways work well I suppose.